Integrations/Cymulate/Cymulate.py

```diff
@@ -0,0 +1,400 @@
+        start_query_time, _ = parse_date_range(first_fetch_time, date_format=DATE_FORMAT, utc=True)
+        start_query_time = ts_add_minutes(start_query_time, 2)
+
+    incidents = []
+
+    # Get the incident from server
+    items = client.get_attack_info(module_type, start_query_time)
+
+    demisto.info("******CYMULATE-LOGS*******")
+    demisto.info(start_query_time)
+    demisto.info(len(items))
+    demisto.info("******CYMULATE-LOGS*******")
+
+    for item in items:
+        if not item['InProgress']:
+            technical_data = client.get_attack_technical_info(module_type, item['Id'])
+            incident_created_time = dateparser.parse(item['Timestamp'])
+
+            for incident in technical_data:
+                if not only_penatrated or incident['Status'] == 'Penetrated':
+                    incident_data = {
+                        'name': item['Name'],
+                        'occurred': incident_created_time.strftime(DATE_FORMAT),
+                        'rawJSON': json.dumps(incident)
+                    }
+
+                    incidents.append(incident_data)
+
+    if len(incidents) > 0:
+        last_incident_ts = incidents[0]['occurred']
+        next_run_ts = (datetime.strptime(last_incident_ts, DATE_FORMAT) + timedelta(minutes=1)).strftime(DATE_FORMAT)
+    else:
+        next_run_ts = start_query_time
+
+    next_run = {"last_fetch": next_run_ts}
+
+    return next_run, incidents[:limit], incidents[limit:]
+
+
+def cymulate_get_incident_info(client):
+    """
+    This function return technical info about incident
+
+    Args:
+        client (Client): Cymulate client
+
+    Returns:
+        Incident contained details technical data
+    """
+
+    # Get module_type
+    module_type = CymulateModuleTypeEnum[demisto.args().get('module_type')]
+
+    # Get incident_id
+    incident_id = demisto.args().get('incident_id')
+
+    # Execute command on API
+    technical_data = client.get_attack_technical_info(module_type, incident_id)
+
+    return technical_data
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    session_token = demisto.params()['x-token']
+    header = {"x-token": session_token}
+
+    # How many time before the first fetch to retrieve incidents
+    fetch_time = demisto.params().get('fetch_time', '3 days').strip()
+
+    # Get the module type
+    module_type = demisto.params().get('module_type', CymulateModuleTypeEnum.IMMEDIATE_THREATS)
+
+    # Fetch only 'penatrated'
+    only_penatrated = demisto.params().get('onlyPenatrated', True)
+
+    # Amount limit of returned incidents
+    fetch_limit = 25
+
+    LOG(f'Command being called is: {demisto.command()}')
+    try:
+        client = Client(
+            base_url=BASE_URL,
+            headers=header,
+            verify=False)
+
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            cymulate_test(client=client)
```

Comment:
if `isFetch` is enabled then the test should verify that fetch is working. It means you should call the same function that you are calling in the fetch_incidents